package.json

```diff
@@ -2,6 +2,9 @@
     "name": "peanut-ui",
     "version": "0.2.0",
     "private": true,
+    "engines": {
+        "node": ">=21.1.0 <22.0.0 || ^18.18.0 || ^20.9.0"
+    },
```

Comment:
First time contributor yay :partying_face:

Reply:
not quite 😄 https://github.com/peanutprotocol/peanut-sdk/pull/125

Reply:
but true for the ui lib :D